### DISCUSSION TITLE
Welcome to Kyndryl Design System Discussions!

### DISCUSSION THREAD

**Discussion by iwhitcomb** [kyndryl-digital/kyndryl-storybook#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gigi0Kiki** [comment]
Hello , I'm Komal Kumari working as a System Administrator in Kyndryl (India). And I'm having a kneen interest in programming field so looking for an opportunity or switch so that i can get to know these things.

Thank You.
